style.css

```diff
@@ -352,50 +376,263 @@ label {
     grid-template-columns: 2fr 2fr;
     justify-content: center;
     margin: 0 auto;
-    ;
   }
 
   h1 {
-      font-size: 8vw;
+    font-size: 8vw;
   }
+
   #headline {
     font-size: 1.5vw;
     line-height: 2;
     height: fit-content;
   }
 
-.hello-text {
-  display: flex;
-  flex-direction: column;
-}
+  .hello-text {
+    display: flex;
+    flex-direction: column;
+    padding-left: 8em;
+  }
 
   .bigimage {
     background-image: url("./images/image-big.png");
-    background-size: 100%;
+    background-size: contain;
+    background-repeat: no-repeat;
   }
 
   .blackBoton {
     width: 254px;
   }
 
+  /* project section descktop */
+
+  .projects {
+    margin-top: 15em;
+  }
+
+  .gallery {
+    display: grid;
+    grid-template-columns: 33% 22% 12% 33%;
+    grid-template-rows: auto auto auto auto;
+    gap: 0;
+  }
+
+  .card {
+    margin: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+  .card4,
+  .card7,
+  .card8 {
+    display: block;
+  }
+
   .project-title {
-    transform:rotate(180)
+    transform: rotate(270deg);
+    margin: 0;
+    line-height: 5;
+    font-size: 150px;
+    width: 100%;
+    height: 100%;
+    background-image: url(images/project/image_dots.png);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
+
+  .card2 {
+    width: 100%;
+    grid-column: 2 / 5;
+    background-color: #1a2236;
+  }
+
+  .card2 img {
+    width: 100%;
+    height: 300px;
+  }
+
+  .card2 .pData {
+    zoom: 200%;
+  }
```

Comment:
Please notice that your cards button should follow the white button style, displayed in Figma 😃:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/201448994-0886f03e-1a81-468f-abcc-1e42d80763f9.png)  | ![image](https://user-images.githubusercontent.com/42799579/201449008-3d83a3eb-84e2-4b2b-a411-bd42d06c9164.png)  |

Reply:
adjusted, thank you